sdk/elixir/runtime/main.go

```diff
@@ -17,13 +18,10 @@ const (
 
 func New(
 	// +optional
-	sdkSourceDir *Directory,
+	sdkSourceDir *dagger.Directory,
 ) *ElixirSdk {
 	if sdkSourceDir == nil {
-		sdkSourceDir = dag.Git("https://github.com/dagger/dagger.git").
-			Branch("main").
-			Tree().
-			Directory("sdk/elixir")
+		sdkSourceDir = dag.CurrentModule().Source().Directory("..")
```

Comment:
You can add `.WithoutDirectory("runtime")` here.

Reply:
Added and also exclude `dagger.json` at this step.